admin_app/src/components/SidebarCommon.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from "react";
+
+import { Close, Send } from "@mui/icons-material";
+import {
+  Box,
+  Checkbox,
+  CircularProgress,
+  FormControlLabel,
+  IconButton,
+  Paper,
+  Typography,
+  TextField,
+} from "@mui/material";
+
+import { useAuth } from "@/utils/auth";
+import { SearchResponseBoxData } from "../app/content/SearchSidebar";
```

Comment:
We shouldn't have low-level components importing from the routes components. One option is to declare a higher level type and `SearchResponseBoxData` and `UDResponseBoxData` inherit / extend it.